app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,84 @@
+class QuestionsController < ApplicationController
+
+  # Called for a GET /questions request.
+  def index
+    @questions = Question.all
+
+    respond_to do |format|
+      format.html
+      format.xml  { render :xml => @questions }
+      format.json  { render :json => @questions }
+    end
```

Comment:
Do we need to do this for all these actions? Can we not do something like https://github.cerner.com/HealtheLife/questionnaire_metadata_service/blob/master/config/routes.rb#L10? Don't we only care about the json response for a microservice?

Reply:
Yes for a microservice a json response is fine. I can do that(from link) if I need json response by default. But here this app has html files too. If I want to get json response, I will append the request with .json.